src/dataexplorer/dataexplorer-application/src/components/Patient/Patient.tsx

```diff
@@ -21,21 +22,30 @@ const Patient = () => {
 
     useEffect(() => {
         if (patientData && patientData.total === 1) {
-            patientData && setName(patientData?.entry[0].resource.name[0].given[0]);
+            const name = `${patientData?.entry[0].resource.name[0].given[0]} ${patientData?.entry[0].resource.name[0].family}`;
+            patientData && setName(name);
```

Comment:
Since we already check that patient data is defined in the if-statement above, I think we can remove change `patientData && setName(name); ` to  just `setName(name)`.

Reply:
Not my change - but I agree, So i have removed this in a followup commit.